CMakeLists.txt

```diff
@@ -83,6 +83,9 @@ include(CheckCXXCompilerFlag)
 include(CMakePushCheckState)
 
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/Modules/)
+if (NOT USE_VCPKG)
+    list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/non-vcpkg-extras/)
+endif()
```

Comment:
The advantage of this is we can use `find_package(SndFile)` instead of 

```
if (USE_VCPKG)
    find_package(SndFile CONFIG)
else()
    find_package(SndFile)
endif()
```

But I'm happy doing either

Reply:
I think we can abstract all of that out to a findX.cmake file though. So the main cmake files should just do find_package(x) and it will just work.